htlcswitch/quiescer.go

```diff
@@ -195,6 +219,10 @@ func (q *QuiescerLive) recvStfu(msg lnwire.Stfu,
 	// If so, we will try to resolve any outstanding StfuReqs.
 	q.tryResolveStfuReq()
 
+	if q.isQuiescent() {
+		q.startTimeout()
```

Comment:
Should we instead start the timeout once either side has initialized the process? i.e., whenever `remoteInit` or `localInit` is set.

Reply:
Great question. The purpose of the timeout is really just to prevent jamming the channel indefinitely. In theory, the difference between what is currently implemented and what you're proposing is somewhat trivial. Further, sending `stfu` doesn't actually impose a constraint on the counterparty, but rather it commits to no longer sending updates itself. Therefore I think leaving it as is is perfectly reasonable and has slightly less implementation complexity -- we don't have to introduce new state to figure out if this is the first time we are trying to start the timeout.